guides/source/form_helpers.md

```diff
@@ -581,25 +581,40 @@ Outputs a select box like:
 
 For each of these helpers, you may specify a date or time object instead of a number as the default value, and the appropriate temporal component will be extracted and used.
 
-Choices from a Collection of Arbitrary Objects
-----------------------------------------------
+Choices from a Collection of Complex Objects
```

Comment:
"Complex Objects" is a bit confusing to me, as I don't know what defines a complex object.
I think in all these cases the collections contain model objects, so maybe:
```suggestion
Choices from a Collection of Model Objects
```

Reply:
Agreed, but I think in most cases it would be a collection of Active Record or Active Model objects anyway. Anyone that sees non Active Record/Model objects as models too will probably understand these objects can be used in the helpers as well.

The example uses an Active Record model as well.
I was trying to think of a non Active Record/Model object that could also be used instead (country?, locale?) but that wouldn't make the example any clearer I think.

Another option could be "Domain Objects" or removing it altogether:
```suggestion
Choices from a Collection of Objects
```

Reply:
An array of strings is a "Collection of Objects" too. :stuck_out_tongue_winking_eye:

How does "Compound Objects" or "Composite Objects" sound to you?

Reply:
Well, you can pass arrays 😄 :
```
  <%= form.collection_select :city_id, [[3, "Berlin"], [1, "Chicago"], [2, "Madrid"]], :first, :last %>
```

I'm not seeing how the following should be seen as a collection of "Compound Objects" or "Composite Objects":
```
  <% Country = Struct.new(:country_code, :name) %>
  <%= form.collection_select :country_code, [Country.new(:us, "United States"), Country.new(:jp, "Japan")], :country_code, :name %>
```

Reply:
To me "Arbitrary Objects" or "Objects" is the clearest.
The API docs doesn't use any adjective either:
https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html#method-i-collection_select

Reply:
Ok, let's leave it as "Arbitrary Objects".  Thank you, @p8! :smile:

Reply:
😄